From 61092ff7f121831c0dbd75b64ce8fcb858fc20d6 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 2 May 2018 15:28:23 +0000
Subject: [PATCH] TRITON-357 sdc-cnapi should log errors encountered when
 posting tasks to clients Reviewed by: Dan McDonald <danmcd@joyent.com>
 Approved by: Trent Mick <trentm@gmail.com>

---
 lib/models/server.js | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/lib/models/server.js b/lib/models/server.js
index 376f4c7..76377cc 100644
--- a/lib/models/server.js
+++ b/lib/models/server.js
@@ -2270,7 +2270,11 @@ function (opts) {
         }
     ],
     function (error) {
-        log.info('done posting task to client');
+        if (error) {
+            log.error({err: error}, 'error posting task to client');
+        } else {
+            log.info('done posting task to client');
+        }
         callback(null, taskstatus);
     });
 
-- 
2.21.0

